build.sbt

```diff
@@ -843,6 +843,23 @@ lazy val zio1Json = (projectMatrix in file("json/zio1-json"))
   .jsPlatform(scalaVersions = scala2 ++ scala3, settings = commonJsSettings)
   .dependsOn(core, jsonCommon)
 
+lazy val tethysJson = (projectMatrix in file("json/tethys-json"))
```

Comment:
this should also be added to `allAggregates`

Reply:
corrected